prometheus-targets.json

```diff
@@ -0,0 +1,20 @@
+[
+    {
+        "labels": {
+            "package": "lighthouse-prater.dnp.dappnode.eth",
+            "service": "beacon-chain.lighthouse-prater.dappnode"
+        },
+        "targets": [
+            "beacon-chain.lighthouse-prater.dappnode:8008"
+        ]
+    },
+    {
+        "labels": {
+            "package": "lighthouse-prater.dnp.dappnode.eth",
+            "service": "validator.lighthouse-prater.dappnode"
+        },
+        "targets": [
+            "validator.lighthouse-prater.dappnode:8008"
+        ]
+    }
+]
```

Comment:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline/729795#729795